docs/usermanual/userman_admin.rst

```diff
@@ -662,8 +662,10 @@ Most common settings
 ^^^^^^^^^^^^^^^^^^^^
 
 * DEBUG = <True or False> (default = False) -- Enables debug mode. In case you run into technical issues, enable this setting before troubleshooting / reporting.
-* CONTENT_ROOT = "<path to desired content folder>" (default=ka-lite/content)
+* CONTENT_ROOT = "<path to desired content folder>" (default=~/.kalite/content)
   This is the path that KA Lite will use to look for KA Lite video files to play.  Change the path to another local directory to get video files from that directory. NB! Directory has to be writable for the user running the server in order to download videos.
+* ASSESSMENT_ITEMS_DATABASE_PATH = "<path to desired content folder>" (default=~/.kalite/content/assessmentitems.sqlite)
```

Comment:
I suggest we remove ASSESSMENT_ITEMS_DATABASE_PATH as a setting, and simply hard-code the path in the DATABASES setting. I don't see a scenario when this should really be user-configurable.